src/riak_core_ring_manager.erl

```diff
@@ -24,22 +24,21 @@
 %%
 %% Numerous processes concurrently read and access the ring in a
 %% variety of time sensitive code paths. To make this efficient,
-%% `riak_core' uses `mochiglobal' which exploits the Erlang constant
-%% pool to provide constant-time access to the ring without needing
-%% to copy data into individual process heaps.
+%% `riak_core' uses `persistent_term' to provide constant-time access
+%% to the ring without needing to copy data into individual process heaps.
 %%
-%% However, updating a `mochiglobal' value is very slow, and becomes slower
```

Comment:
maybe just link here? 

http://erlang.org/doc/man/persistent_term.html#description the first paragraph could be copied here to have something before following the link, but removing the previous mochiglobal comments which seem to sort of apply but not sure all apply